src/func.d

```diff
@@ -3235,13 +3241,37 @@ public:
         return true;
     }
 
-    // Determine if function needs
-    // a static frame pointer to its lexically enclosing function
+    /****************************************
+     * Determine if function needs a runtime frame pointer to
```

Comment:
The correct jargon is `static frame pointer` that points to the lexically enclosing function's frame. A `dynamic frame pointer` points to the caller's frame.